data/hai/unfettered side missions.txt

```diff
@@ -147,6 +147,127 @@ mission "Unfettered: Untrustable"
+				to display
+					not "flagship model: Carrier"
+			`As you finish the landing procedure you a warning siren triggers on the bridge. One of your officers warns you that a Lightning Bug is headed straight towards you, at full speed!`
+				to display
+					has "flagship model: Carrier"
+			`	Fortunately it stops as soon as it enters scanning range, and begins a full sweep of your ship, before hailing you. You immediately recognize Mera. "Hey there, <first>! I was warned about a Carrier arriving," she looks to her side, probably at one member of her crew as she continues, "because someone thought it was the Giftbringer..." before looking back to you, with a smile, "Anyway, what a pleasant surprise to know it was you. I hope you dont mind me taking some scans, you'll be compensated for it naturally. In fact, I'll allow you to come with me to the lab and choose any item you desire!"`
+			choice
+				`	"That seems like a good deal, I'm in!"`
+					goto deal
+				`	"You can do better than scan that ship, in fact I'm willing to sell it to you."`
+					goto sell
+				`	"My privacy is very important to me. Stop that scan at once."`
+			`	She seems really surprised, but gives a sign to her officers to stop the scanning. "In that case we will erase the information collected. Truly I had no idea you would see it this way. Still, this is such a shame..."`
+				decline
+			label deal
+			action
+				set "unfettered: carrier scans"
+			`	The scan is finished in a record speed, and Mera invites you on her ship to get to the lab. When you arrive on the bridge she is alone. "I like to be able to speak freely, you see," she says as you look at the empty bridge. "You never know who is listening," she says as she sets a course. "I have something to show you. But first, tell me, how did you acquire this Carrier? We Unfettered like to share stories, sometimes around campfires..." she says before clapping her hands. Suddenly the atmosphere of the room changes, and you can see the outside of the ship as if it was no longer there.`
+			choice
+				`	"There is not much to say, I simply bought it."`
+					goto bought
+				`	"I acquired it in a grand battle for freedom!"`
+					goto freedom
+				`	"How did you do that?"`
+					goto explain
+				`	"I prefer not to talk about it."`
+			`	"Suit yourself."`
+				goto testingsite
+			label bought
+			`	"I see you are very humble, captain."`
+				goto testingsite
+			label freedom
+			`	"So you are already Unfettered in your heart!" She sighs. "I hope it means you understand that we do not all desire battle, but we do see it as a necessity."`
+				goto testingsite
+			label explain
+			`	"This?" She says, looking around. "It's just a few projectors, in truth. This was not their primary intended use, however," she says with a chuckle.`
+			label testingsite
+			`	You quickly arrive at a collection of craters, which remind you of the moon. She stops there and takes a deep breath.`
```

Comment:
Do we know if FL has ever been to the moon? Also, in this context, it should probably be capitalized, or better, `Earth's moon`. (Though most moons have craters, so idk.)